src/main/groovy/wooga/gradle/build/unity/tasks/UnityBuildEnginePlayerTask.groovy

```diff
@@ -0,0 +1,74 @@
+package wooga.gradle.build.unity.tasks
+
+import org.gradle.api.file.RegularFileProperty
+import org.gradle.api.provider.Property
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.InputFile
+import org.gradle.api.tasks.Optional;
+
+
+class UnityBuildEnginePlayerTask extends AbstractUnityBuildEngineTask {

```

Comment:
Similar to above, I would name this PlayerBuildEngineUnityTask: "This is an Unity Task for a player build"

Reply:
Done.